src/shared/components/K8sNameInput/K8sNameInput.js

```diff
@@ -28,21 +29,21 @@ export const K8sNameInput = ({
     inputRef,
     onChange,
   });
-
+  if (!props.readOnly) delete props.readOnly;
   const { className, compact, ...inputProps } = props || {};
 
-  const inputClassName = classnames('fd-input', className, {
-    'fd-input--compact': compact,
+  const inputClassName = classnames('input-full', className, {
+    'ui5-content-density-compact': compact,
   });
 
   const input = (
-    <FormInput
-      ref={_ref}
-      type="text"
+    <Input
+      // ref={_ref}
```

Comment:
ref doesn't exist in UI5, so we can potentially remove all of the refs for input components